frontend/packages/dev-console/src/components/import/serverless-function/AddServerlessFunctionForm.tsx

```diff
@@ -120,44 +139,117 @@ const AddServerlessFunctionForm: React.FC<
     }
   }, [image, isPipelineEnabled]);
 
+  const ExtensionCard: React.FC<ExtensionCardProps> = ({ icon, link, title, description }) => {
+    return (
+      <Card>
+        <CardTitle>
+          <div className="odc-serverless-extensions-card-logo">
+            <div className="odc-serverless-extensions-card-logo__icon">
+              <span className="co-catalog-item-icon__bg">
+                <img
+                  className="co-catalog-item-icon__img co-catalog-item-icon__img--large"
+                  src={icon}
+                  aria-hidden
+                  alt=""
+                />
+              </span>
+            </div>
+            <div className="odc-serverless-extensions-card-logo__name">
+              <h1 className="odc-serverless-extensions-card-logo__name__link">
+                <ExternalLink href={link} text={title} />
+              </h1>
+              <span className="odc-serverless-extension-provider text-muted">
+                {t('devconsole~Provided by Red Hat')}
+              </span>
+            </div>
+          </div>
+        </CardTitle>
+        <CardBody>
+          <p>{description}</p>
+        </CardBody>
+      </Card>
+    );
+  };
+
   return (
     <form onSubmit={handleSubmit} data-test="create-serverless-function-form">
-      <FormBody>
-        <GitSection builderImages={builderImages} />
-        {showFullForm && (
-          <>
-            {builderImages && <ServerlessFunctionStrategySection builderImages={builderImages} />}
-            <AppSection
-              project={values.project}
-              noProjectsAvailable={projects.loaded && _.isEmpty(projects.data)}
-            />
-            {showPipelineSection && <PipelineSection builderImages={builderImages} />}
-            <AdvancedSection values={values} />
-          </>
-        )}
-        {validated !== ValidatedOptions.default &&
-          strategy !== ServerlessBuildStrategyType.ServerlessFunction && (
-            <Alert
-              variant="warning"
-              isInline
-              title={t('devconsole~func.yaml is not present or builder strategy is not s2i')}
+      <Flex direction={{ default: 'column', sm: 'row' }}>
+        <FlexItem flex={{ default: 'flex_1' }} alignSelf={{ default: 'alignSelfFlexStart' }}>
```

Comment:
![Screenshot from 2023-06-15 09-10-16](https://github.com/openshift/console/assets/139310/2bde8a23-1734-457b-9976-844c88c5fc75)

Esp. with the light theme I notice these different gaps between the boxes.

You can remove the default spacing from the Flex components with the `spaceItems` props:

```suggestion
      <Flex direction={{ default: 'column', sm: 'row' }} spaceItems={{ default: 'spaceItemsNone' }}>
        <FlexItem flex={{ default: 'flex_1' }} alignSelf={{ default: 'alignSelfFlexStart' }}>
```

Wdyt?